sdk/protocol-manager-poc/src/implementation/aaveV3Plugin.ts

```diff
@@ -0,0 +1,167 @@
+                    }
+                } catch (e) {
+                    console.log("error in collateral loop", e)
+                    continue;
+                }
+            }
+
+            const debts: Record<AddressValue, AaveV3PoolDebtConfig> = {}
+            for (const asset of filteredAssetsList) {
+                const { token: quoteToken, config: { borrowingEnabled, reserveFactor }, caps: { borrowCap }, data: { totalVariableDebt, totalStableDebt, variableBorrowRate } } = asset;
+                // TODO: Remove Try/Catch once PriceService updated to use protocol oracle
+                if (quoteToken.symbol === TokenSymbol.WETH) {
+                    // WETH can be used as collateral on AaveV3 but not borrowed.
+                    continue;
+                }
+
+                try {
+                    const RESERVE_FACTOR_TO_PERCENTAGE_DIVISOR = 10000n
+                    const PRECISION_PRESERVING_OFFSET = 1000000n
+                    const RATE_DIVISOR_TO_GET_PERCENTAGE = Number((PRECISION_PRESERVING_OFFSET - 100n).toString())
+
+                    const rate = Number(((variableBorrowRate * PRECISION_PRESERVING_OFFSET) / PRESISION_BI.RAY).toString()) / RATE_DIVISOR_TO_GET_PERCENTAGE
+                    const totalBorrowed = totalVariableDebt + totalStableDebt
+                    debts[quoteToken.address.value] = {
+                        token: quoteToken,
+                        // TODO: If we further restricted pools we could have token pair prices
+                        price: Price.createFrom({
+                            baseToken: quoteToken,
+                            quoteToken: poolBaseCurrencyToken,
+                            value: new BigNumber(asset.price.toString()).toString(),
+                        }),
+                        priceUSD: Price.createFrom({
+                            baseToken: quoteToken,
+                            quoteToken: CurrencySymbol.USD,
+                            value: new BigNumber(asset.price.toString()).toString(),
+                        }),
+                        rate: Percentage.createFrom({ percentage: rate }),
+                        totalBorrowed: tokenAmountFromBaseUnit({token: quoteToken, amount: totalBorrowed.toString() }),
+                        debtCeiling: TokenAmount.createFrom({token: quoteToken, amount: borrowCap === 0n ? UNCAPPED_SUPPLY : borrowCap.toString() }),
+                        debtAvailable: tokenAmountFromBaseUnit({token: quoteToken, amount: borrowCap === 0n ? UNCAPPED_SUPPLY : (borrowCap - totalBorrowed).toString() }),
+                        dustLimit: tokenAmountFromBaseUnit({token: quoteToken, amount: '0' }),
+                        originationFee: Percentage.createFrom({ percentage: Number((reserveFactor / RESERVE_FACTOR_TO_PERCENTAGE_DIVISOR).toString()) }),
+                        borrowingEnabled
+                    }
+                } catch (e) {
+                    console.log("error in debt loop", e)
```

Comment:
I don't think we should have console.log at this level. If something can throw we should either:
- Capture it and try to recover
- Let it raise further and be catched by the user of the plugin (ProtocolsManager) which should handle it properly

Reply:
Yea, sorry was using this to siliently ignore before implementing oracle asset prices fully. Forgot to remove